src/wasm.h

```diff
@@ -1522,6 +1522,7 @@ class Rethrow : public SpecificExpression<Expression::RethrowId> {
 // 'throw_ref' from the new EH proposal
 class ThrowRef : public SpecificExpression<Expression::ThrowRefId> {
 public:
+  ThrowRef() {}
```

Comment:
```suggestion
  ThrowRef() = default;
```
Nit: It looks other classes in this file use this